client/layout/masterbar/logged-in.jsx

```diff
@@ -94,31 +93,23 @@ class MasterbarLoggedIn extends Component {
 		 *
 		 * This code makes it possible to reset the failed migration state when clicking My Sites too.
 		 */
-		if ( config.isEnabled( 'tools/migrate' ) ) {
-			const { migrationStatus, currentSelectedSiteId } = this.props;
+		const { migrationStatus, currentSelectedSiteId } = this.props;
 
-			if ( currentSelectedSiteId && migrationStatus === 'error' ) {
-				/**
-				 * Reset the in-memory site lock for the currently selected site
-				 */
-				this.props.updateSiteMigrationMeta( currentSelectedSiteId, 'inactive', null );
+		if ( currentSelectedSiteId && migrationStatus === 'error' ) {
+			/**
+			 * Reset the in-memory site lock for the currently selected site
+			 */
+			this.props.updateSiteMigrationMeta( currentSelectedSiteId, 'inactive', null );
 
-				/**
-				 * Reset the migration on the backend
-				 */
-				requestHttpData(
-					'site-migration',
-					http( {
-						apiNamespace: 'wpcom/v2',
-						method: 'POST',
-						path: `/sites/${ currentSelectedSiteId }/reset-migration`,
-						body: {},
-					} ),
-					{
-						freshness: 0,
-					}
-				);
-			}
+			/**
+			 * Reset the migration on the backend
+			 */
+			wpcom.req
+				.post( {
+					path: `/sites/${ currentSelectedSiteId }/reset-migration`,
+					apiNamespace: 'wpcom/v2',
+				} )
+				.catch( () => {} );
```

Comment:
I guess we need the `catch()` because `requestHttpData` was swallowing errors and passing them to the data layer meta?

Reply:
Yes, it would store the `error` to `http-data` state here: https://github.com/Automattic/wp-calypso/blob/trunk/client/state/data-layer/http-data.ts#L117